lib/modules/home/components/slide_item.dart

```diff
@@ -0,0 +1,89 @@
+part of '../home_module.dart';
+
+class SlideItem extends StatelessWidget {
+  final SurveyInfo survey;
+  const SlideItem({
+    Key? key,
+    required this.survey,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      fit: StackFit.expand,
+      children: [
+        Bone.hidden(
+          child: Image(
+            image: NetworkImage(survey.coverImageUrl!),
+            fit: BoxFit.fill,
```

Comment:
the background image is scaled a bit, we can make a refactor to change to `BoxFit.cover` for keeping the correct scale ratio 😄

Reply:
@manh-t it's weird because it show perfectly in ios side
Also from their document
```dart
enum BoxFit {
  /// Fill the target box by distorting the source's aspect ratio.
  ///
  /// ![](https://flutter.github.io/assets-for-api-docs/assets/painting/box_fit_fill.png)
  fill,
```